wbooks/App.js

```diff
@@ -16,19 +16,19 @@ import {
 } from 'react-native';
 
 import BookList from './src/app/screens/booklist';
-import styles from './styles';
+import {createAppContainer} from 'react-navigation';
+import {createStackNavigator} from 'react-navigation-stack';
+import BookDetail from './src/app/screens/book-detail';
 
-const App: () => React$Node = () => {
-  return (
-    <>
-      <StatusBar  />
-      <SafeAreaView>
-        <View style={styles.container}>
-          <BookList />
-        </View>
-      </SafeAreaView>
-    </>
-  );
-};
+const MainNavigator = createStackNavigator({
+  Home: {screen: BookList},
+  Details: {screen: BookDetail},
+},
+{
+  initialRouteName: 'Home',
```

Comment:
if you have a file named routes you should changes this string for the constant. if you dont just leave it that way